dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/metric_config_panel_item.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState, useContext } from 'react';
+import {
+  EuiTitle,
+  EuiComboBox,
+  EuiSpacer,
+  EuiButton,
+  EuiFieldText,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiPanel,
+  EuiText,
+} from '@elastic/eui';
+import { useDispatch } from 'react-redux';
+import { change as changeVizConfig } from '../../../../../redux/slices/viualization_config_slice';
+import {
+  AGGREGATIONS,
+  GROUPBY,
+  METRICS_AGGREGATION_OPTIONS,
+  numericalTypes,
```

Comment:
Change numericalTypes to NUMERICAL_TYPES. This change has been merged to main. Please check.